sentry_dart_plugin/bin/sentry_dart_plugin.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:sentry_dart_plugin/sentry_dart_plugin.dart';
+
+void main(List<String> arguments) {
```

Comment:
```suggestion
Future<void> main(List<String> arguments) {
```

I think it also needs an `exit(0);` after running `SentryDartPlugin().run(arguments);`

Reply:
not sure since it dies automatically when the process lifecycle is over